apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -167,10 +167,9 @@ function LandingPage({
           },
         });
 
+        setLoadingWorkshopsAsParticipant(false);
```

Comment:
I think with the nesting this won't actually turn off the loader if an error shows since if the `fetch` hits an error it will jump to the `catch` block rather than finishing the `try` block I believe. I think it may require a `setLoadingWorkshopsAsParticipant(false);` line in both the `response.ok` block and the `catch` block